pkg/broker/receiver.go

```diff
@@ -179,24 +181,40 @@ func (r *Receiver) sendEvent(ctx context.Context, tctx cloudevents.HTTPTransport
 		return nil, err
 	}
 
+	// Set up the metrics context
+	ctx, _ = tag.New(ctx,
+		tag.Insert(TagTrigger, trigger.String()),
+		tag.Insert(TagBroker, fmt.Sprintf("%s/%s", trigger.Namespace, t.Spec.Broker)),
+	)
+	// Record event count and filtering time
+	startTS := time.Now()
+	defer func() {
+		dispatchTimeMS := int64(time.Now().Sub(startTS) / time.Millisecond)
+		stats.Record(ctx, MeasureTriggerDispatchTime.M(dispatchTimeMS))
```

Comment:
Do we need to count TriggerEventsTotal separately, or can we get the count from the distribution?

Reply:
True, the histogram emits a `_count` metric that will always have the same value as `trigger_events_total`:

```
trigger_dispatch_time_bucket
trigger_dispatch_time_sum
trigger_dispatch_time_count
```

We could eliminate the `trigger_events_total` metric. Should we then rename `trigger_dispatch_time` to just `trigger_dispatch`?

```
trigger_dispatch_bucket
trigger_dispatch_sum
trigger_dispatch_count
```